[FIX] Better message for not determined journal.

NL66278 opened the pull request.

NL66278 committed:
[FIX] Better message for not determined journal.

StefanRijnhart reviewed (changes requested):
Looks like you need to adapt the test to your modification.

NL66278 committed:
[FIX] Only check first (and constant) part of message in text.

Test on import bank statement checked exception message for wrong journal. Part of message has become variable,
so test had to be adapted.

NL66278 commented:
@StefanRijnhart Test has been adapted.

StefanRijnhart reviewed (approved):
Green!
